src/main/webapp/adminOnly/AdminOnly.js

```diff
@@ -184,7 +185,7 @@ class AdminOnly extends Component {
     form.investigatorFirstName = this.state.formData.investigatorFirstName;
     form.investigatorLastName = this.state.formData.investigatorLastName;
     form.initialDate = this.parseDate(this.state.formData.initialDate);
-    form.initialReviewType = this.state.formData.initialReviewType;
+    form.initialReviewType = JSON.stringify(this.state.formData.initialReviewType);
```

Comment:
Curious about why this needs to be stringified - usually that's for display purposes. I'm pretty sure it doesn't do anything structurally to the data here.

Reply:
Select component returns selected value as an json object { label:"xxxx", value: "zzz"}. If we store only selectedValue.value then we need to recreate selected Value at read. We saved as stringified, we parse at read.

Reply:
@rushtong can we merge this PR?

Reply:
Absolutely ... I approved it!